api-lib/node-fetch-shim.ts

```diff
@@ -0,0 +1,8 @@
+import fetch from 'node-fetch';
+
+// This injects fetch into the global context for node.
+if (!globalThis.fetch) {
+  globalThis.fetch = fetch as unknown as typeof globalThis.fetch;
```

Comment:
do we still need the `@types/node-fetch` changes from ba558f9c? if not, can they be removed?

Reply:
That was done wrong, I made a mistake merging it, @jsur put that in the generated code, if we want it in a place like that it should be in a place like `api-lib/Gql.ts` but I want to refactor all of that

Reply:
sorry, i should have been more specific i guess. i meant specifically adding `@types/node-fetch` to `devDependencies` in package.json. if that's not needed, let's remove it